src/bam/record.rs

```diff
@@ -414,12 +481,8 @@ impl Record {
                 ),
             }
         };
-
-        if ret < 0 {
-            Err(AuxWriteError::Some)
-        } else {
-            Ok(())
-        }
+        
+        Ok(())
```

Comment:
Why removing the error handling here?

Reply:
restored the error handling. In htslib 1.7, aux_append returns void, so we're going to need to drop this eventually.